R/github-utils.R

```diff
@@ -16,7 +34,7 @@ github_owner <- function() {
   github_origin()[["owner"]]
 }
 
-github_source <- function() {
+github_owner_upstream <- function() {
```

Comment:
This name makes much more sense to me.

Reply:
We have `pr_pull_source()`. Should this be `github_owner_source()`? Or `github_source_owner()`?

Reply:
You renamed `pr_pull_source()` to `pr_pull_upstream()` in a separate PR (which I merged very recently):

Rename `pr_pull_source()` to `pr_pull_upstream()` #617

Reply:
Oh I guess I must be using an old version locally 🤪